### ISSUE TITLE
Always pulls new image regardless of setting on properties when running on Windows

### ISSUE THREAD

**Issue by jmaes12345** [localstack/localstack-java-utils#61]
Using the annotation `@LocalstackDockerProperties` we should be able to set "pullNewImage" to false - indeed it should be false by default.

However, I have found running this on Windows always attempts to pull a new image, regardless of the setting.

I believe this is due to the check for whether the image exists within `cloud.localstack.docker.Container` line 71 checks if the list of images contains the image name but does not account for Windows line endings within the `ListImagesCommand.execute()` method as the list is split on `\n` only, leaving a `\r` at the end of the name which is not on the expected `fullImageName`.

I only noticed this was the expected behaviour when I was trying to work around [this](https://github.com/localstack/localstack/issues/4115) issue and noticed that the option didn't work!

**Steps to reproduce:**
* Create a JUnit class annotated with:
```
@LocalstackDockerProperties(
		imageTag = "0.12.11",
		pullNewImage = false,
		services = { "s3" }
)
```
* Run the JUnit

**Expected:**
The log message below is not produced:
```
cloud.localstack.docker.Container createLocalstackContainer
INFO: Pulling latest image...
```

**Actual:**
It is.

**Comment by whummer** [comment]
Thanks for reporting @jmaes12345 - this should be fixed in #65 . We have just released a new version `0.2.13` to Maven Central. Please let us know if the problem persists. Thanks!

**Comment by jmaes12345** [comment]
I just tried it with 0.2.13, and can confirm that it's fixed 😄 
Thanks @pinzon and @whummer !
